# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../iptraf/destdir.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- iptraf-3.0.0/src/Makefile.vanilla	2007-08-15 13:26:02.000000000 +0200
+++ iptraf-3.0.0/src/Makefile	2007-08-15 13:26:19.000000000 +0200
@@ -145,7 +145,7 @@
 # installation rule
 
 install:
-	@./install.sh $(TARGET) $(WORKDIR) $(LOGDIR) $(LOCKDIR)
+	@./install.sh $(DESTDIR)$(TARGET) $(DESTDIR)$(WORKDIR) $(DESTDIR)$(LOGDIR) $(DESTDIR)$(LOCKDIR)
 
 # Upgrade rule
 #
